myplugin/content/example/templates/example/index.html

```diff
@@ -154,15 +146,24 @@ <h3>1. Create an AppTemplate</h3>
                     </li>
                 </ul>
             </li>
-            <li><strong>Write Configuration Script:</strong>
+            <li><strong>Configure Security Groups</strong>
                 <ul>
-                    <li>Step 1: Select the checkbox to automatically generate the SSH setup for users. This
-                        will create a script that sets up SSH access for the students.
+                    <li>Select one or more existing security groups to be used for the AppTemplate. These
+                        security groups define the network access rules for the instances launched from this
+                        AppTemplate.
                     </li>
-                    <li>Step 2: Enter instantiation and account attributes to be used in the configuration
+                </ul>
+            </li>
+            <li><strong>Write Configuration Script:</strong>
+                <ul>
+                    <li>Step 1: Enter instantiation and account attributes to be used in the configuration
                         script. For example <code>username:password</code> as account attributes.
```

Comment:
Hier sollte das eigentlich verändert werden
```suggestion
                        script. For example <code>username,password</code> as account attributes.
```